network-api/networkapi/wagtailpages/pagemodels/dear_internet.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+
+from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
+from wagtail.core.models import Page
+from wagtail.core.fields import StreamField
+
+from . import customblocks
+from .mixin.foundation_metadata import FoundationMetadataPageMixin
+from ..utils import set_main_site_nav_information
+
+
+class DearInternetPage(FoundationMetadataPageMixin, Page):
+    intro_text_1 = models.CharField(
+        max_length=500,
+        help_text='Intro text 1',
+    )
+
+    intro_text_2 = models.CharField(
+        max_length=500,
+        help_text='Intro text 2',
+    )
+
+    intro_text_3 = models.CharField(
+        max_length=500,
+        help_text='Intro text 3',
+    )
+
+    intro_text_4 = models.CharField(
+        max_length=500,
+        help_text='Intro text 4',
+    )
```

Comment:
This feels like not a great solution... what does each of these texts represent? Because if we make this a streamfield for paragraphs of text, then content editors can just make sure they add four paragraphs and then we can use the same `{% for block in page.intro_texts %}` that we already use for letters. No need to hardcode "four" in this way. @kristinashu would that be an option?

Reply:
Let's wait for design to chime in. From the comps it looks like we designed the page with 4 intro blocks + illustrations around them.

Reply:
updated to use StreamField!